run file through eslint, and remove arrow functions

gavinsharp opened the pull request:
Attempting to get our app to run on IE11 - this is a requirement. I "transpiled" the `SessionAccount` class using [buble](https://buble.surge.sh/).

ademidun commented on lib/sessionaccount.js:
```diff
@@ -1,359 +1,310 @@
 
 
-        this.auth.userhandler = {
-            onSuccess: (result) => {
```
I thought the benefit of a compiler like [buble](https://buble.surge.sh/) was that you don't have to remove arrow functions in your actual code. Won't it be automatically removed when you build the project?

gavinsharp replied:
Discussed in person, but just for posterity: it didn't seem worth it to try to set up a transpiling (e.g. babel) deploy pipeline for both of these small libraries right now, but longer-term it definitely makes sense to do that (particularly if they accumulate more code).

gavinsharp commented:
As I updated this to also update to the latest [`properly-util-js`](https://github.com/GoProperly/properly-util-js/) (which includes https://github.com/GoProperly/properly-util-js/pull/1), I also updated a few dependencies (`url-parse` and `uuid`) to resolve npm audit warnings.
